awscli/customizations/cloudformation/artifact_exporter.py

```diff
@@ -331,7 +331,6 @@ class GraphQLSchemaResource(Resource):
     # Necessary to support Definition
     PACKAGE_NULL_PROPERTY = False
 
-
```

Comment:
Let's keep the extra newline. For Python, classes should be separated by two newlines.